core/internal/features_test.go

```diff
@@ -983,6 +984,11 @@ func TestIntegration_FluxMonitor_NewRound(t *testing.T) {
 	require.NoError(t, err)
 	job.Initiators[0].InitiatorParams.Feeds = cltest.JSONFromString(t, fmt.Sprintf(`["%s"]`, mockServer.URL))
 	job.Initiators[0].InitiatorParams.PollingInterval = models.MustMakeDuration(15 * time.Second)
+	// NOTE: Must set a very large idle timeout here because log.StartedAt is
```

Comment:
We would have to change it to something dynamic otherwise we'll always need a really large idle timeout. Are you suggesting converting to a factory and setting it dynamically based on time.Now?

Reply:
Yeah, I think I'd said that exact thing on the last version of this PR. I'm fine leaving it as is for now.